packages/esm-generic-patient-widgets-app/src/resources/useObs.ts

```diff
@@ -54,6 +84,18 @@ export interface UseObsResult {
 
 type ObsResult = FHIRResource['resource'] & {
   conceptUuid: string;
+  dataType: string;
+  valueString?: string;
+  valueCodeableConcept?: ObsCodedData;
+};
+
+type ObsCodedData = {
+  coding: Array<ObsCodedCodingData>;
+};
+
+type ObsCodedCodingData = {
```

Comment:
We already have `FHIRCode` defined in the framework.

Reply:
Should I add the property `display?: string;` to the `FHIRCode`?

Reply:
Yes, that would be a good idea!